src/tangle/sources/mod.rs

```diff
@@ -52,5 +55,5 @@ pub trait InputSource {
         index: MilestoneIndex,
     ) -> Result<BoxStream<Result<BlockData, Self::Error>>, Self::Error>;
 
-    async fn ledger_updates(&self, index: MilestoneIndex) -> Result<LedgerUpdateStore, Self::Error>;
+    fn ledger_updates(&self, index: MilestoneIndex) -> LedgerUpdateStore;
```

Comment:
Why no async?

Reply:
Could not get it to compile yet, but working on it.

Reply:
I don't know what happened, compiled without problems.